SU-HSDO/suhumsci
# feat(STN-782) Limited width color bands

# READY FOR REVIEW

## Summary
As a Developer, Site Owner, or Contributor, I want a limited width option for Color Band that only spans the center column.

## Need Review By (Date)
asap

## Urgency
medium

## Steps to Test
1. npm run test
2. import the config
3. create a flexible page with color bands, limited and full width
4. Make sure both options work as expected

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
